packages/kotti-ui/source/kotti-split-button/types.ts

```diff
@@ -0,0 +1,51 @@
+import { yocoIconSchema } from '@3yourmind/yoco'
+import { z } from 'zod'
+
+import { KottiButton } from '../kotti-button/types'
+import { createLooseZodEnumSchema } from '../zod-utilities/enums'
+
+export module KottiSplitButton {
+	export enum Type {
+		DEFAULT = KottiButton.Type.DEFAULT,
+		PRIMARY = KottiButton.Type.PRIMARY,
+	}
+	export const typeSchema = createLooseZodEnumSchema(Type)
+
+	export const Size = {
+		...KottiButton.Size,
```

Comment:
because of this confusion, you tried to re-export the enum but you couldn't destruct an enum inside so you made it a const

and the util we have relies on actually having a native `enum`